----------------------------------------------------------------
2013-08-01 10:39:32.227 GMT:
 Booting Derby version The Apache Software Foundation - Apache Derby - 10.5.3.0 - (802917): instance a816c00e-0140-3976-081e-0000039ddff8
on database directory /tmp/jetty-ode.war-_ode-any-/webapp/WEB-INF/jpadb  

Database Class Loader started - derby.database.classpath=''

------------  BEGIN SHUTDOWN ERROR STACK -------------

ERROR XSLAQ: cannot create log file at directory /tmp/jetty-ode.war-_ode-any-/webapp/WEB-INF/jpadb/log.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.getLogDirectory(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.getControlFileName(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.checkpointWithTran(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.checkpoint(Unknown Source)
	at org.apache.derby.impl.store.raw.RawStore.stop(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.stop(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.shutdown(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.shutdown(Unknown Source)
	at org.apache.derby.impl.db.DatabaseContextImpl.cleanupOnError(Unknown Source)
	at org.apache.derby.iapi.services.context.ContextManager.cleanupOnError(Unknown Source)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.cleanupOnError(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection40.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver40.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(Unknown Source)
	at org.apache.ode.il.dbutil.Database.shutdown(Database.java:122)
	at org.apache.ode.axis2.ODEServer.shutDown(ODEServer.java:343)
	at org.apache.ode.axis2.hooks.ODEAxisServlet.stop(ODEAxisServlet.java:68)
	at org.apache.axis2.engine.ListenerManager.stop(ListenerManager.java:153)
	at org.apache.axis2.engine.ListenerManager$ListenerManagerShutdownThread.run(ListenerManager.java:243)


------------  END SHUTDOWN ERROR STACK -------------


------------  BEGIN SHUTDOWN ERROR STACK -------------

ERROR XSLAQ: cannot create log file at directory /tmp/jetty-ode.war-_ode-any-/webapp/WEB-INF/jpadb/log.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.getLogDirectory(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.getControlFileName(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.checkpointWithTran(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.checkpoint(Unknown Source)
	at org.apache.derby.impl.store.raw.RawStore.stop(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.stop(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.shutdown(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.shutdown(Unknown Source)
	at org.apache.derby.impl.db.DatabaseContextImpl.cleanupOnError(Unknown Source)
	at org.apache.derby.iapi.services.context.ContextManager.cleanupOnError(Unknown Source)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.cleanupOnError(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection40.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver40.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(Unknown Source)
	at org.apache.ode.il.dbutil.Database.shutdown(Database.java:122)
	at org.apache.ode.axis2.ODEServer.shutDown(ODEServer.java:343)
	at org.apache.ode.axis2.hooks.ODEAxisServlet.stop(ODEAxisServlet.java:68)
	at org.apache.axis2.engine.ListenerManager.stop(ListenerManager.java:153)
	at org.apache.axis2.engine.ListenerManager$ListenerManagerShutdownThread.run(ListenerManager.java:243)


------------  END SHUTDOWN ERROR STACK -------------


------------  BEGIN SHUTDOWN ERROR STACK -------------

ERROR XSLAQ: cannot create log file at directory /tmp/jetty-ode.war-_ode-any-/webapp/WEB-INF/jpadb/log.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.getLogDirectory(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.getControlFileName(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.checkpointWithTran(Unknown Source)
	at org.apache.derby.impl.store.raw.log.LogToFile.checkpoint(Unknown Source)
	at org.apache.derby.impl.store.raw.RawStore.stop(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.stop(Unknown Source)
	at org.apache.derby.impl.services.monitor.TopService.shutdown(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.shutdown(Unknown Source)
	at org.apache.derby.impl.db.DatabaseContextImpl.cleanupOnError(Unknown Source)
	at org.apache.derby.iapi.services.context.ContextManager.cleanupOnError(Unknown Source)
	at org.apache.derby.impl.jdbc.TransactionResourceImpl.cleanupOnError(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection40.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver40.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(Unknown Source)
	at org.apache.ode.il.dbutil.Database.shutdown(Database.java:122)
	at org.apache.ode.axis2.ODEServer.shutDown(ODEServer.java:343)
	at org.apache.ode.axis2.hooks.ODEAxisServlet.stop(ODEAxisServlet.java:68)
	at org.apache.axis2.engine.ListenerManager.stop(ListenerManager.java:153)
	at org.apache.axis2.engine.ListenerManager$ListenerManagerShutdownThread.run(ListenerManager.java:243)


------------  END SHUTDOWN ERROR STACK -------------


2013-08-01 10:54:40.918 GMT:
Shutting down instance a816c00e-0140-2fd2-2e43-000003fcf610
----------------------------------------------------------------
